builtin/providers/aws/resource_aws_s3_bucket_test.go

```diff
@@ -131,16 +131,19 @@ func testAccCheckAWSS3BucketWebsite(n string, indexDoc string, errorDoc string)
 		}
 
 		if *out.IndexDocument.Suffix != indexDoc {
-			return fmt.Errorf("bad: %s", out.IndexDocument)
+			if out.IndexDocument.Suffix != nil {
+				return fmt.Errorf("bad index document suffix: %s", *out.IndexDocument.Suffix)
+			}
+			return fmt.Errorf("bad index document suffix, is nil")
```

Comment:
I didn't think IndexDocument could be nil, so I didn't add the if statement here, but it could be nil in the future when we add support for redirects.